core/src/test/java/com/google/cloud/sql/core/CloudSqlConnectorRegistryTest.java

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2016 Google Inc.
+ * Copyright 2023 Google LLC
```

Comment:
These can stay as is, since we're only moving the file.

Reply:
Sadly, since this is a new file, the header-check requires this year's date.